### PR TITLE
Pure baseline rerun (seed variance measurement)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
We need another baseline rerun to better understand seed variance on the current code. Previous reruns (senku yoo7f9hc: 2.26, frieren gl3gofxt: 2.31) show variance. A third data point helps calibrate what's noise vs signal.

## Instructions
NO changes. Run unmodified current noam:
`python train.py --agent gilbert --wandb_name "gilbert/baseline-v3" --wandb_group pure-baseline-rerun`

## Baseline: val/loss=2.2068
---
## Results

**W&B run:** oy1745vi
**Best epoch:** 67 / 100 (30-min timeout)
**Peak memory:** ~10.5 GB

### Metrics

| Split | val/loss | mae_surf_Ux | mae_surf_Uy | mae_surf_p |
|---|---|---|---|---|
| val_in_dist | 1.6426 | 0.302 | 0.182 | 22.3 |
| val_tandem_transfer | 3.2166 | — | — | 40.7 |
| val_ood_cond | 1.9102 | — | — | 21.0 |
| val_ood_re | 18869.6 | — | — | 31.1 |
| **combined val/loss (3-split)** | **2.2565** | | | |

**Baseline val/loss: 2.2068 → This run: 2.2565 (+0.049)**

### What happened

Third baseline data point confirms substantial seed variance in the current setup.

All four reruns:
| Run | Agent | val/loss |
|---|---|---|
| original baseline | — | 2.2068 |
| yoo7f9hc | senku | 2.26 |
| oy1745vi (this) | gilbert | 2.2565 |
| gl3gofxt | frieren | 2.31 |

Range: 2.2068 – 2.31 (~0.10 spread). This means any improvement smaller than ~0.05 is within single-run noise and should not be trusted from a single run. Results landing in the 2.25–2.26 range are essentially equivalent to the best baseline.

Practically, this calibrates the signal threshold: improvements of ~0.05+ are potentially real, while improvements < 0.03 are likely noise. The recent merged experiments (e.g. EMA start epoch #858) that showed ~0.04–0.05 improvements sit right at the noise boundary.

### Suggested follow-ups

- **Use multi-seed averaging** when a result lands within 0.05 of baseline to confirm it is a real improvement.
- **The effective baseline is ~2.26 ± 0.05** — frame all future comparisons against this range rather than the single best run (2.2068).

### CODE REVIEW

**Comment by tcapelle** [comment]
## Review — Closing
Baseline rerun v3: val_loss=2.2565 (+2.3%). Four baseline reruns now: 2.2068, 2.26, 2.31, 2.26. The ±4% variance confirms our noise floor. Closing.

**Comment by morganmcg1** [comment]
ADVISOR: EP5 Gate Check — PASS

**PR #898 (5L STRING GradNorm + Y-sym, long run)**
W&B run: `ylrp8f97`

| Epoch | Steps | val_primary | Gate | Status |
|-------|-------|-------------|------|--------|
| EP1 | 5,493 | 11.002% | ≤30% | ✅ PASS |
| EP2 | 10,987 | 8.067% | ≤16% | ✅ PASS |
| EP3 | 16,481 | 7.558% | ≤8.0% | ✅ PASS |
| EP4 | 21,975 | 7.323% | ≤7.5% | ✅ PASS |
| EP5 | 27,469 | **7.217%** | ≤7.5% | ✅ PASS (margin 0.28pp) |

Excellent clean progression. PR#898 is the frontrunner: 7.217% at EP5 is ahead of PR#899 (7.350%) and well below the gate. The GradNorm + Y-sym + 5L combo is converging cleanly.

**Next gate:** EP10 (step ~54,938): val_primary ≤ 7.2%

Continue the run. Report at EP10 with per-channel breakdown (especially vol_p — that's the metric we're watching closely).
